torch/csrc/jit/tensorexpr/schedule.h

```diff
@@ -22,10 +22,12 @@ class TORCH_API LoopNest {
 
   std::vector<For*> getLoopStmtsFor(Tensor*) const;
   Stmt* getLoopBodyFor(Tensor*) const;
+  bool hasLoopBodyFor(Tensor*) const;
   std::unordered_map<Tensor*, Stmt*> tensor_to_stmt_;
 
   void Vectorize(Stmt*);
   void ComputeInline(Stmt* s);
+  void ComputeInlineWithRandom(Stmt* s);
```

Comment:
Minor: I am a bit split whether this should be exposed as part of the Schedule, or as part of implementation details within ComputeInline.

Reply:
I had this thought too.  Would be open to changing it down the line.